Gruntfile.js

```diff
@@ -239,7 +240,7 @@ module.exports = function(grunt) {
 
     githooks: {
       all: {
-        'pre-commit': 'jshint cover'
+        'pre-commit': 'jshint'
```

Comment:
Sure.. my initial thought was, if you want to remove jQuery migrate at some point, you could just revert this commit. Now with this being in the same commit it would potentially have additional side effects.